packages/page-treasury/src/Overview/Tip.tsx

```diff
@@ -157,6 +157,13 @@ function Tip ({ bestNumber, className = '', hash, isMember, members, tip }: Prop
           />
         )}
       </td>
+      <td className='mini ui--media-1000'>
```

Comment:
We need this is the header as well. (Should also adjust the classes, maybe 1200 or 1400 until we can sort out the busy nature)

Reply:
ah right totally missed it.

Reply:
Don't worry, I miss that a lot since they stuff is in 2 places - so don't feel all alone.